src/main/java/com/cburch/logisim/gui/main/Canvas.java

```diff
@@ -442,7 +442,7 @@ public String getToolTipText(MouseEvent event) {
     boolean showTips = AppPreferences.COMPONENT_TIPS.getBoolean();
     if (showTips) {
       Canvas.snapToGrid(event);
-      Location loc = Location.create(event.getX(), event.getY());
+      Location loc = Location.create(event.getX(), event.getY(), false);
```

Comment:
`var` or `final var` please?

Reply:
Although this is not related to this PR (only a fix) I changed it. I see that there is still work to do in "cleaning up" the code, as in the same section there are a lot of situations without `var` and `final var`. Maybe we should define a strategy?

Reply:
I think that technically all newly code should use `var` and `final var` whenever possible. All the old code should be corrected separately to keep scope of each PR as narrow as possible. This makes reviewing such PR much easier.